src/utils/ensure-base-exists.utils.ts

```diff
@@ -88,8 +84,16 @@ export const ensureBaseTestsExists = async ({
     return { shaToCompareAgainst: base };
```

Comment:
Note: there is a risk we could get a chain of waits, which could burn a lot of GH minutes